lmi_utils/label_utils/csv_to_coco.py

```diff
@@ -196,27 +291,63 @@ def visualize(self, polygon, fname):
         im[~mask] *= 0.25
         cv2.imshow('plot',im.astype(np.uint8))
         cv2.waitKey(100)
-                
+        
+    
+def copy_images_in_folder(path_img, path_out, fnames=None):
+    """
+    copy the images from one folder to another
+    Arguments:
+        path_img(str): the path of original image folder
+        path_out(str): the path of output folder
+    """
+    os.makedirs(path_out, exist_ok=True)
+    if not fnames:
+        l = glob.glob(os.path.join(path_img, '*.png')) + glob.glob(os.path.join(path_img, '*.jpg'))
+    else:
+        l = [f"{path_img}/{fname}" for fname in fnames]
+    for f in l:
+        shutil.copy(f, path_out)
 
 if __name__ == '__main__':
     import argparse
     ap = argparse.ArgumentParser()
     ap.add_argument('--path_imgs', required=True, help='the path to the images')
     ap.add_argument('--path_csv', default='labels.csv', help='[optinal] the path of a csv file that corresponds to path_imgs, default="labels.csv" in path_imgs')
-    ap.add_argument('--classes', required=True, help='the class categories in the dataset using comma to separate each category')
```

Comment:
the classes command line arguments is for extracting bboxes from a subset of classes. It would be better to keep/improve it.